core/modules/query/Predicate.h

```diff
@@ -36,18 +36,14 @@
 
 // Local headers
 #include "query/typedefs.h"
-#include "query/BoolTerm.h"
+#include "query/BoolFactorTerm.h"
```

Comment:
Please, move this include one line up to comply with the required sorting order!